### PR TITLE
Fix date selector update when scrolling

### PR DESCRIPTION
**Pull request by ptbrowne**
The date selector was not updating because transactions rows refs were not passed to the TopMost instance : while scrolling, we could not figure out the highest visible row.

### CODE REVIEW

File: src/ducks/transactions/TransactionRow.jsx [hunk 1]
```diff
@@ -32,8 +32,14 @@ const sAmount = styles['bnk-op-amount']
 const sAction = styles['bnk-op-action']
 
 class _RowDesktop extends React.PureComponent {
-  onSelectTransaction = () =>
+  constructor(props) {
+    super(props)
+    this.onSelectTransaction = this.onSelectTransaction.bind(this)
```
Comment (drazik) [inline src/ducks/transactions/TransactionRow.jsx]: Why do you need to change this? Is there a problem with class properties and fat arrows?

File: src/components/List/index.jsx [hunk 2]
```diff
@@ -8,8 +8,10 @@ export const Header = ({ children }) => (
   <div className={styles['c-list-header']}>{children}</div>
 )
 
-export const Row = ({ children }) => (
-  <div className={styles['c-list-row']}>{children}</div>
+export const Row = props => (
+  <div ref={props.onRef} className={styles['c-list-row']}>
+    {props.children}
+  </div>
```
Comment (drazik) [inline src/components/List/index.jsx]: Nit: it seems it's a good use case for [`React.forwardRef`](https://reactjs.org/docs/forwarding-refs.html)

Comment (Crash--) [reply src/components/List/index.jsx]: WHAT! Thank you! Didn't know about this one.

File: src/ducks/transactions/TransactionRow.jsx (hunk 1 above)
Comment (ptbrowne) [reply src/ducks/transactions/TransactionRow.jsx]: Yes, it is not possible to mock fat arrow methods in tests.

Comment (ptbrowne) [reply src/ducks/transactions/TransactionRow.jsx]: https://github.com/airbnb/enzyme/issues/365#issuecomment-217368222

**Pull request merged**